sentry.client.config.ts

```diff
@@ -12,17 +12,22 @@ Sentry.init({
   dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
 
   // Adjust this value in production, or use tracesSampler for greater control
-  tracesSampleRate: 1,
+  tracesSampleRate: ["development", "heroku"].includes(process.env.NODE_ENV)
+    ? 1.0
+    : 0.1,
 
   // Setting this option to true will print useful information to the console while you're setting up Sentry.
-  debug: true,
+  debug: ["development", "heroku"].includes(process.env.NODE_ENV),
 
   replaysOnErrorSampleRate: 1.0,
 
   // This sets the sample rate to be 10%. You may want this to be 100% while
   // in development and sample at a lower rate in production
-  replaysSessionSampleRate:
-    process.env.NEXT_PUBLIC_NODE_ENV === "development" ? 1.0 : 0.1,
+  replaysSessionSampleRate: ["development", "heroku"].includes(
+    process.env.NODE_ENV
```

Comment:
I assume this is OK converting from `NEXT_PUBLIC_NODE_ENV` to `NODE_ENV`?

Reply:
No that's not right, thanks for catching.